### PR TITLE
Козлова Екатерина. Вариант 28. SEQ. Повышение контраста полутонового изображения посредством линейной растяжки гистограммы.

### PR DESCRIPTION
**Pull request by KatyaKozlova**
Полутоновое изображение представлено вектором целых чисел. Значения пикселей изображения масштабируются на весь диапазон от 0 до 255. Для этого находятся минимальное и максимальное значения пикселей, которые используются как конечные точки для растяжки, распределяя все пиксели между этими границами.

### CODE REVIEW

File: tasks/seq/kozlova_e_contrast_enhancement/func_tests/main.cpp [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  test_task_sequential.PostProcessing();
+
+  EXPECT_EQ(out[0], 12);
+  EXPECT_EQ(out[1], 0);
+  EXPECT_EQ(out[2], 63);
+  EXPECT_EQ(out[3], 127);
+  EXPECT_EQ(out[4], 255);
+}
+
+TEST(kozlova_e_contrast_enhancement_seq, test_image2) {
```
Comment (Gn4ik) [inline tasks/seq/kozlova_e_contrast_enhancement/func_tests/main.cpp:49]: add test for negative values

**Review by Gn4ik — CHANGES_REQUESTED** [review]

**Review by Gn4ik — APPROVED** [review]

**Review by ElizavetaFrolova — APPROVED** [review]

**Review by aobolensk — APPROVED** [review]

File: tasks/seq/kozlova_e_contrast_enhancement/func_tests/main.cpp [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  ASSERT_EQ(test_task_sequential.Validation(), true);
+  test_task_sequential.PreProcessing();
+  test_task_sequential.Run();
+  test_task_sequential.PostProcessing();
+
+  EXPECT_EQ(in, out);
+}
+
+TEST(kozlova_e_contrast_enhancement_seq, test_difference_input) {
+  std::vector<int> in{10, 20, 30, 100, 200, 250};
```
Comment (allnes) [inline tasks/seq/kozlova_e_contrast_enhancement/func_tests/main.cpp:110]: please parametrize height and width in image and check it

**Review by allnes — APPROVED** [review]
